qa/tasks/cephfs/test_mirroring.py

```diff
@@ -613,20 +613,20 @@ def test_cephfs_mirror_stats(self):
         second = res[TestMirroring.PERF_COUNTER_KEY_NAME_CEPHFS_MIRROR_PEER][0]
         self.assertGreater(second["counters"]["snaps_synced"], first["counters"]["snaps_synced"])
         self.assertGreater(second["counters"]["last_synced_start"], first["counters"]["last_synced_start"])
-        self.assertGreater(second["counters"]["last_synced_end"], second["counters"]["last_synced_start"])
+        self.assertGreaterEqual(second["counters"]["last_synced_end"], second["counters"]["last_synced_start"])
```

Comment:
Remind me please - how granular is this particular stat? (I remember it to be seconds granular).

Reply:
OK. Let me put this to test today.